README.md

```diff
@@ -0,0 +1,105 @@
+
+## Local Development
+
+### Prerequisites
+
+* [Go](https://golang.org/) 1.24.3 or later
+* [Docker](https://www.docker.com/products/docker-desktop/) to spin up required databases
+* [devbox](https://github.com/jetify-com/devbox) (optional but recommended) - starts a shell with all required tools
+* [Maven](https://maven.apache.org/) for building the Java-based scheduler component
+* [direnv](https://direnv.net/) (optional) to automatically configure the development environment
+
+### Make Targets
+
+| Target                                                                | Description                                                                |
+|-----------------------------------------------------------------------|----------------------------------------------------------------------------|
+| `make build`                                                          | Build all components                                                       |
+| `make generate-fakes`                                                 | Generate test mocks/fakes                                                  |
+| `make generate-openapi-generated-clients-and-servers`                 | Generate clients and servers from OpenAPI specs                            |
+| `make test`                                                           | Run unit tests against PostgreSQL                                          |
+| `make clean && make test POSTGRES_TAG=x.y`                            | Run unit tests against specific PostgreSQL version                         |
+| `make integration`                                                    | Run integration tests against PostgreSQL                                   |
+| `make clean && make integration POSTGRES_TAG=x.y`                     | Run integration tests against specific PostgreSQL version                  |
+| `make acceptance-tests`                                               | Run acceptance tests (see [acceptance/README.md](acceptance/README.md))    |
+| `make lint`                                                           | Check code style                                                           |
+| `OPTS=--fix make lint`                                                | Check code style and apply auto-fixes                                      |
+| `make fmt`                                                            | Format Go code                                                             |
+| `make clean`                                                          | Remove build artifacts and generated code                                  |
+| `make mta-build`                                                      | Build MTA archive for deployment                                           |
+| `make mta-deploy`                                                     | Deploy to Cloud Foundry using MTA                                          |
+
+### Database Setup
```

Comment:
I would move the database setup section after the running tests section,

Running tests targets will automatically spin up the postgres db via docker